src/engine/srv.c

```diff
@@ -296,6 +296,45 @@ dss_rpc_hdlr(crt_context_t *ctx, void *hdlr_arg,
 	return sched_req_enqueue(dx, &attr, real_rpc_hdlr, rpc);
 }
 
+static int
+secondary_rpc_hdlr(crt_context_t *ctx, void *hdlr_arg, void (*real_rpc_hdlr)(void *), void *arg)
+{
+	struct dss_xstream	*primary_dx;
+	crt_rpc_t		*rpc = (crt_rpc_t *)hdlr_arg;
+	uint32_t		 tag;
+	int			 xs_id, rc;
+
+	D_DEBUG(DB_TRACE, "Received secondary RPC, opc: %#x\n", rpc->cr_opc);
+
+	rc = crt_req_dst_tag_get(rpc, &tag);
+	if (rc) {
+		D_ERROR("Failed to get tag from RPC, "DF_RC"\n", DP_RC(rc));
+		return rc;
+	}
+
```

Comment:
might be good to print here tag as debug before starting processing/error checking

Reply:
Sure, I'll add it in follow-on patch.